core/src/main/java/hudson/model/Slave.java

```diff
@@ -306,6 +306,8 @@ public void setLabelString(String labelString) throws IOException {
         return new GetClockDifference1();
     }
 
+    @Override
+    @Nonnull
```

Comment:
I agree. Better to just remove since the parent Javadoc describes the `null` case in a portable way